spec/core/ExpectationResultSpec.js

```diff
@@ -73,4 +73,30 @@ describe("buildExpectationResult", function() {
     var result = jasmineUnderTest.buildExpectationResult({actual: 'some-value'});
     expect(result.actual).toBe('some-value');
   });
+
+  it("handles nodejs assertions", function() {
+    var assert = require('assert');
+    var error;
+    var value = true;
+    var expectedValue = false;
+    try { assert.equal(value, expectedValue) } catch(e) { error = e; }
+
+    expect(Object.keys(error)).toContain(
+      'code', 'actual', 'expected', 'operator'
+    );
+    expect(error.code).toEqual('ERR_ASSERTION');
+    expect(error.expected).toEqual(expectedValue);
+    expect(error.actual).toEqual(value);
+    expect(error.operator).toEqual('==');
```

Comment:
This block looks like we're testing the `assert` functionality itself and not necessarily how Jasmine handles it. Is this necessary?

Reply:
Actually, we're checking for undocumented, but fairly stable internals of third party library. If something will ever change in nodejs assertion library, we will know.

Reply:
I think I would still rather not test the runtime here, but this is fair safeguarding for undocumented behavior.

We definitely need to check that Jasmine is putting values in the correct place on the `result` below.